certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -120,12 +121,12 @@ def test_manual_dns_auth(context):
         '--manual-cleanup-hook', context.manual_dns_cleanup_hook,
         '--pre-hook', 'echo wtf.pre >> "{0}"'.format(context.hook_probe),
         '--post-hook', 'echo wtf.post >> "{0}"'.format(context.hook_probe),
-        '--renew-hook', 'echo renew >> "{0}"'.format(context.hook_probe)
+        '--renew-hook', 'echo deploy >> "{0}"'.format(context.hook_probe)
```

Comment:
Can we leave this at "renew"?

If for some reason we want to change this to "deploy", I think we should also change the `assert_hook_execution` call down below to "deploy" as well.

Reply:
Original tests were using the `deploy` keyword. But it is just a naming convention, without functional impact. I change the word, it makes more sense